src/routes/roomTypes.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: ao
+ * Date: 2018-11-12
+ * Time: 12:34 AM
+ */
+
+use \Psr\Http\Message\ServerRequestInterface as Request;
+use \Psr\Http\Message\ResponseInterface as Response;
+
+// Get All room types for a hotel
+$app->get('/api/hotels/{id}/room-types', function (Request $request, Response $response, array $args) {
+
+    $hotel_id = $args['id'];
+
+    $sql = "SELECT * FROM room_type where hotel_id = '$hotel_id' ";
+    try {
+        // Get DB Object
+        $db = new db();
+        // Connect
+        $db = $db->connect();
+        $stmt = $db->query($sql);
+        $customers = $stmt->fetchAll(PDO::FETCH_OBJ);
+        $db = null;
+
+        return $response->write(json_encode($customers));
+    } catch (PDOException $e) {
+
+        $db = null;
+
+        $response->write($e);
+        return $response->withStatus(500);
+    }
+});
+
+// create a room type to a hotel
+$app->post('/api/hotels/{id}/room-types', function (Request $request, Response $response, array $args) {
+
+    $hotel_id = $args['id'];
+
+    $sql = "INSERT INTO `room_type` (`id`, `type_name`, `occupancy`, `description`, `price`, `available_slots`, `hotel_id`, `total_slots`)" .
+        " VALUES (NULL, :type_name, :occupancy, :description, :price, :slots, :hotel_id, :slots);";
+
+    $parsedBody = $request->getParsedBody();
+
+    $db = new db();
+    $db = $db->connect();
+
+    try {
+        $stmt = $db->prepare($sql);
+        $stmt->bindParam(":type_name", $parsedBody["type_name"]);
+        $stmt->bindParam(":occupancy", $parsedBody["occupancy"]);
+        $stmt->bindParam(":description", $parsedBody["description"]);
+        $stmt->bindParam(":price", $parsedBody["price"]);
+        $stmt->bindParam(":slots", $parsedBody["slots"]);
+        $stmt->bindParam(":hotel_id", $hotel_id);
+
+        if ($stmt->execute()) {
+            $insertedId = $db->lastInsertId();
+
+            $db = null;
+
+            $responseArray = array();
+            $responseArray["id"] = $insertedId;
+            return $response->write(json_encode($responseArray))->withStatus(200);
+        }
+    } catch (PDOException $e) {
+
+        $db = null;
+
+        $response->write($e);
+        return $response->withStatus(500);
+    }
+});
+
+// update a room type
+$app->put('/api/room-types/{id}', function (Request $request, Response $response, array $args) {
```

Comment:
is `xxx-yyy` a thing in url? shouldn't it be better with camel case?

Reply:
in my understanding, the url should avoid capitalized letter usage

Reply:
then lets use `-`. I dont like camel case in URL